pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -377,6 +377,10 @@ func (o *Options) Run() error {
 			fmt.Fprintf(o.Out, "Upgradeable=False\n\n  Reason: %s\n  Message: %s\n\n", c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  "))
 		}
 
+		if c := findClusterOperatorStatusCondition(cv.Status.Conditions, "ReleaseAccepted"); c != nil && c.Status == configv1.ConditionFalse {
```

Comment:
I'm thinking we want to simply display `ReleaseAccepted` status, not just when it's failed. It may not be set though so something like CVO does https://github.com/openshift/cluster-version-operator/blob/484ebf2c299bdf03764ce18974525c8276507740/pkg/cvo/status.go#L395

/cc @LalatenduMohanty 
/cc @wking

Reply:
Or trigger on !=True, so we display on Unknown?  I dunno if we should distract users in the happy =True case

Reply:
Ah, I forgot about the `configv1.ConditionUnknown` status, thanks :+1:.

I am leaning more towards informing the users only when things don't go as planned in this case: triggering on `ReleaseAccepted != True`. I would guess that the average user doesn't know about the update being a two-phase process (loading, applying). In the case of `ReleaseAccepted == True`, things go well as always (as before introducing the `ReleaseAccepted` to the CVO). So the printing could be a little bit  distracting maybe?

But I have no problem printing the `ReleaseAccepted` condition whenever running `oc adm upgrade`, not just when it's failed. It is an interesting piece of information.

Reply:
Just go with showing the user when "things don't go as planned"

Reply:
I have changed the logic to trigger the print on `ReleaseAccepted != True` in the new commit https://github.com/openshift/oc/commit/bf761c3443dcd88934d08ef87cf9a8f270d74b27.